### PR TITLE
[MRG] add Windows build

### PR DESCRIPTION
**Pull request by massich**

### CODE REVIEW

**Comment by massich** [comment]
Its green on appveyor and travis (-linux) , I'll assume that travis (-mac) is working

File: .appveyor.yml [hunk 1]
```diff
@@ -45,21 +55,45 @@ install:
       }
       $env:LD_MKL_PATH='C:\Program Files (x86)\IntelSWTools\compilers_and_libraries_2018.0.124\windows\mkl\lib\intel64_win\'
       $env:LIB="$env:LD_MKL_PATH;$env:LIB"
-  - nuget install openblas -o C:\
+      $env:MKLROOT='C:\Program Files (x86)\IntelSWTools\compilers_and_libraries_2018.0.124\windows\mkl'
 
+  ############################################################################
+  # Install OpenBlAS
+  ############################################################################
+  - nuget install openblas -o C:\
```
Comment (agramfort) [inline .appveyor.yml:63]: this is not tested yet.

**Comment by massich** [comment]
No. I think I removed, it was there just to have `cblas.h` I think that this is somewhat relatet to what @xoviat was asking to openblas repo.

**Comment by massich** [comment]
No. I think I removed, it was there just to have `cblas.h` I think that this is somewhat related to what @xoviat was asking to openblas repo.

**Comment by massich** [comment]
No. I think I removed, it was there just to have `cblas.h` I think that
this is somewhat related to what @xoviat was asking to openblas repo.

**Pull request merged**